packages/react/src/components/List/ListItem/ListItem.jsx

```diff
@@ -39,6 +39,7 @@ const ListItemPropTypes = {
   expanded: PropTypes.bool,
   value: PropTypes.string.isRequired,
   /** string value or callback render function */
```

Comment:
Need to move this comment down below. This new value was inserted in between.

Reply:
Comment moved and doc added for new property